### PR TITLE
Added recipe for daviddrysdale/python-phonenumbers

### PR DESCRIPTION
**Pull request by pmlandwehr**
This is a [Python port](https://github.com/daviddrysdale/python-phonenumbers) of Google's [libphonenumber](https://github.com/googlei18n/libphonenumber)

### CODE REVIEW

**Comment by patricksnape** [comment]
Looks fine to me.

File: recipes/phonenumbers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - phonenumbers
+    - phonenumbers.carrierdata
+    - phonenumbers.data
+    - phonenumbers.geodata
+    - phonenumbers.shortdata
+    - phonenumbers.tzdata
+
+about:
+  home: https://github.com/daviddrysdale/python-phonenumbers
+  license: Apache Software License
```
Comment (jakirkham) [inline recipes/phonenumbers/meta.yaml]: Please update with version (e.g. `Apache X.Y`).

**Comment by pmlandwehr** [comment]
(Close/reopen to force a Travis CI and appveyor rebuild. Accidentally killed the former, not sure why the latter died.)

**Comment by patricksnape** [comment]
@jakirkham What is the current guidelines around the `python-` prefix?

File: recipes/phonenumbers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{% set version = "7.4.1" %}
+
+package:
+  name: phonenumbers
+  version: {{ version }}
+
+source:
+  fn: phonenumbers-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/source/p/phonenumbers/phonenumbers-{{ version }}.tar.gz
```
Comment (patricksnape) [inline recipes/phonenumbers/meta.yaml]: I think we are preferring `https://pypi.io/` rather than `https://files.pythonhosted.org`

**Comment by pmlandwehr** [comment]
Just wanted to ping folks on this repo - the merge has been accepted, but no `phonenumbers-feedstock` has shown up. Any ideas where it's gone?

**Comment by jakirkham** [comment]
Good question. Could you please open a new PR with this recipe? We will happily merge it. Though we are rate-limited ATM. So, it won't be converted a feedstock for a little bit. Hope that's ok. Sorry for the inconvenience.

**Comment by jakirkham** [comment]
So, this is pretty strange. I think this branch got reused for the `pykalman` PR ( https://github.com/conda-forge/staged-recipes/pull/734 ). As `phonenumbers` was removed in that PR, it was removed here too. This all happened before it was merged. In short, this only appears to have been merged because PR ( https://github.com/conda-forge/staged-recipes/pull/734 ) was a direct descendant that was merged. I'll try to resurrect phonenumbers from the history and re-add it for you, @pmlandwehr. Will ping you in the new PR.

**Comment by jakirkham** [comment]
Re-added in this PR ( https://github.com/conda-forge/staged-recipes/pull/728 ).

**Pull request merged**